uitools/examples/cpp_quick/src/demos/OverviewMapDemo.qml

```diff
@@ -0,0 +1,52 @@
+// Copyright 2021 ESRI
+//
+// All rights reserved under the copyright laws of the United States
+// and applicable international laws, treaties, and conventions.
+//
+// You may freely redistribute and use this sample code, with or
+// without modification, provided you include the original copyright
+// notice and use restrictions.
+//
+// See the Sample code usage restrictions document for further information.
+//
+
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import Esri.ArcGISRuntime 100.13
+import Esri.ArcGISRuntime.Toolkit 100.13
+import DemoApp 1.0
+
+DemoPage {
+    sceneViewContents: Component {
+        SceneView {
+            id: view
+            OverviewMap {
+                anchors {
+                    top: parent.top
+                    left: parent.left
+                    margins: 10
+                }
+                geoView: parent
+            }
+            OverviewMapDemoModel {
+                geoView: view
+            }
+        }
+    }
+    mapViewContents: Component {
+        MapView {
+            id: view
+            OverviewMap {
+                anchors {
+                    top: parent.top
+                    left: parent.left
+                    margins: 10
+                }
+                geoView: parent
+            }
+            OverviewMapDemoModel {
```

Comment:
💭 I'm not quite sure that the term "model" is right for these types in a MVC sense... they know about the controller and are present in the ui layer which is not what i'd expect from a model.

Reply:
I'm happy to go through and change the suffixes about. I'm not convinced they are _not_ models but I understand the concern. 

All the QML files could have the name `Form` appended, and `Model` could be removed from the C++ files, this would be closer to the templates, I think.